peewee_async/pool.py

```diff
@@ -77,6 +85,70 @@ async def create(self) -> None:
         )
 
 
+class Psycopg3PoolBackend(PoolBackend):
+    """Asynchronous database connection pool based on psycopg + psycopg_pool libraries.
+    """
+
+    async def create(self) -> None:
+        """Create connection pool asynchronously.
+        """
+
+        pool = psycopg_pool.AsyncConnectionPool(
+            format_dsn(
+                'postgresql',
+                host=self.connect_params['host'],
+                port=self.connect_params['port'],
+                user=self.connect_params['user'],
+                password=self.connect_params['password'],
+                path=self.database,
+            ),
+            min_size=self.connect_params.get('minsize', 1),
+            max_size=self.connect_params.get('maxsize', 20),
+            max_lifetime=self.connect_params.get('pool_recycle', 60 * 60.0),
+            open=False,
+            kwargs={
+                'cursor_factory': psycopg.AsyncClientCursor,
+                'autocommit': True,
+            }
+        )
+
+        await pool.open()
+        self.pool = pool
+
+    def has_acquired_connections(self) -> bool:
+        if self.pool is not None:
+            return self.pool._nconns - self.pool._num_pool > 0
+        return False
+
+    async def acquire(self) -> ConnectionProtocol:
+        """Acquire connection from pool.
+        """
+        if self.pool is None:
+            await self.connect()
+        assert self.pool is not None, "Pool is not connected"
+        return await self.pool.getconn()
```

Comment:
Found some stats will not count since we use getconn instead of connection asynccontextmanager. https://github.com/psycopg/psycopg/blob/master/psycopg_pool/psycopg_pool/pool_async.py#L198

Reply:
I created an issue #300 No need to fix in the PR